### DISCUSSION TITLE
Is this repo still maintained?

### DISCUSSION THREAD

**Discussion by mariomurrent-softwaresolutions** [react-native-elements/react-native-elements#3925] [General]
Is this repo still maintained?

**Comment by ChromeQ** [comment]
Seems like this is dead, what do others recommend as an alternative?

**Reply by mariomurrent-softwaresolutions** [reply]
I switched to react-native-paper

**Reply by XuJin186** [reply]
react-native-paper  Also stopped maintenance.

**Reply by mariomurrent-softwaresolutions** [reply]
Really? Haven't seen anything like that in the Repo/Issues List

**Comment by vincicat** [comment]
Someone mentioned https://react-native-velements.thevikalp.com is still actively maintained and this is a fork from RNE.

remarks:
* React Native Paper is still "alive" -- they have a new release recently, but they are Material Design which may not suit your iOS app
* You may try UI kitten / Gluestack UI / RNUI (by wix, [link](https://wix.github.io/react-native-ui-lib/)) if you want something not so iOS and android - but why are you doing this when you are using RN?

**Comment by Monte9** [comment]
It wasn't for a while but we're working on reviving it again! Stay tuned

**Reply by ChromeQ** [reply]
Good to hear @Monte9, are we talking days, weeks or months eta?

**Comment by lucksp** [comment]
According to this discord post, RNE will be coming back online

https://discord.com/channels/1036635878728146984/1036635880791736433/1392734078872326175

**Comment by andysteinwachs** [comment]
@Monte9 Can we get an update here, please? Will this come back and continue to be maintained?

**Reply by theianmay** [reply]
Hello! I am a new contributor. I know other contributors have been working on making progress towards getting this back into a working state. I would have to defer to @Monte9 for timeline details. We are also trying to keep folks updated via our Discord - link here: https://discord.com/invite/e9RBHjkKHa

**Comment by theianmay** [comment]
## 🚀 React Native Elements v5 Beta is Live!  

Hello everyone! 👋  

**v5.0.0-beta.1** is now available! This is a **beta release**, intended for testing and feedback. The **full stable v5.0.0** is still in the pipeline.  

---
###⚠️ What you need to know
- This is not yet stable, intended for testing and feedback.
- For example, a known issue is: Button disabled state styling on Android. Workarounds and discussion available here: #3985
- Associated issue: https://github.com/facebook/react-native/issues/53849
- You can install the beta via NPM:
```bash
npm install @rneui/base@beta @rneui/themed@beta
```
---

### 📝 Feedback
Please provide **bugs, suggestions, or feedback**:  
- GitHub Discussion: https://github.com/react-native-elements/react-native-elements/discussions/4007 
- Discord: https://discord.com/invite/e9RBHjkKHa

**Next Steps:**  
- I will be closing this for now to keep things organized.
- All ongoing beta feedback should go to the Discussion or Discord.
